examples/kubernetes/cilium-config.yaml

```diff
@@ -0,0 +1,43 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: cilium-config
+  namespace: kube-system
+data:
+  # If you do not want to run cilium in debug mode change it to false
+  debug: "true"
```

Comment:
Let's move this down and default to `false`

Reply:
Fixed